docs/milestones.md

```diff
@@ -0,0 +1,16 @@
+# Milestones
+
+* [[M0] Estructura de datos para organizar información que permite 1."Representar un año concreto" y 2."Definir los parámetros relevantes de ese año( existencias iniciales, evolución de precios, producción y precipitaciones)"](https://github.com/joseantonio2001/OlivApp/milestone/5).
```

Comment:
Una estructura de datos no es un producto. Por favor, revisa la presentación y el guión del objetivo.

Reply:
Podría ser un módulo que contenga la estructura de datos para organizar la información de los años?? Así dicho de manera coloquial

Reply:
Podría ser... un producto. Creo que en las últimas modificaciones del guión de este objetivo lo he dejado bastante claro. No tratéis de buscar "lo que hay que poner". Tratad de entender qué es lo que es un producto, y qué no.

Reply:
Okey ya he aplicado los cambios y corregido lo comentado, listo para revisión.